### ISSUE TITLE
FNX-5056 ⁃ Find in page a11y

### ISSUE THREAD

**Issue by boek** [mozilla-mobile/fenix#148]
Parent #118

**Comment by eeejay** [comment]
The next/prev and close buttons should have larger touch targets.

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on the latest Nightly build 190806 with Sony Xperia Z5 Premium (Android 7.1.1), and LG G7 Fit (Android 8.1).
